### PR TITLE
Update private preview note for cmdlets

### PR DESCRIPTION
**Pull request by bishalgautam-ms**
In this PR, we are:
1. updating Preview note for the cmdlets which are still in private-preview mode.
2. updating example company-name to the company name 'contoso'

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Review by dariomws — APPROVED** [review]
@bishalgautam-ms, maybe you meant "Public Preview"? I see these cmdlets available in the public preview version (3.0.1-preview).

**Comment by bishalgautam-ms** [comment]
> @bishalgautam-ms, maybe you meant "Public Preview"? I see these cmdlets available in the public preview version (3.0.1-preview).

Yes, you are right.

**Comment by dariomws** [comment]
@bishalgautam-ms, are you planning to update the PR? Thanks!

**Comment by bishalgautam-ms** [comment]
> @bishalgautam-ms, are you planning to update the PR? Thanks!

All the changes are made for this PR.
Thanks @dariomws for asking.

**Review by get-itips — COMMENTED** [review]
@bishalgautam-ms

Looks like these cmdlets are in the Public Preview version of the module, shouldn't the note say Public preview instead of private preview?

Thank you

File: teams/teams-ps/teams/Get-CsTeamsShiftsConnectionOperation.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ schema: 2.0.0
 
 ## SYNOPSIS
 
+**Note:** This cmdlet is currently in private preview.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsTeamsShiftsConnectionOperation.md:16]:
```suggestion
**Note:** This cmdlet is currently in public preview.
```

File: teams/teams-ps/teams/Get-CsTeamsShiftsConnectionSyncResult.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ schema: 2.0.0
 
 ## SYNOPSIS
 
+**Note:** This cmdlet is currently in private preview.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsTeamsShiftsConnectionSyncResult.md:16]:
```suggestion
**Note:** This cmdlet is currently in public preview.
```

File: teams/teams-ps/teams/Get-CsTeamsShiftsConnectionWfmTeam.md [hunk 3]
```diff
@@ -13,6 +13,8 @@ schema: 2.0.0
 
 ## SYNOPSIS
 
+**Note:** This cmdlet is currently in private preview.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsTeamsShiftsConnectionWfmTeam.md:16]:
```suggestion
**Note:** This cmdlet is currently in public preview.
```

File: teams/teams-ps/teams/Get-CsTeamsShiftsConnectionWfmUser.md [hunk 4]
```diff
@@ -13,6 +13,8 @@ schema: 2.0.0
 
 ## SYNOPSIS
 
+**Note:** This cmdlet is currently in private preview.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsTeamsShiftsConnectionWfmUser.md:16]:
```suggestion
**Note:** This cmdlet is currently in public preview.
```

File: teams/teams-ps/teams/New-CsTeamsShiftsConnectionBatchTeamMap.md [hunk 5]
```diff
@@ -11,8 +11,11 @@ schema: 2.0.0
 
 # New-CsTeamsShiftsConnectionBatchTeamMap
 
+
 ## SYNOPSIS
 
+**Note:** This cmdlet is currently in private preview.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsShiftsConnectionBatchTeamMap.md:17]:
```suggestion
**Note:** This cmdlet is currently in public preview.
```

File: teams/teams-ps/teams/New-CsTeamsShiftsConnectionInstance.md [hunk 6]
```diff
@@ -13,6 +13,8 @@ schema: 2.0.0
 
 ## SYNOPSIS
 
+**Note:** This cmdlet is currently in private preview.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsShiftsConnectionInstance.md:16]:
```suggestion
**Note:** This cmdlet is currently in public preview.
```

File: teams/teams-ps/teams/New-CsTeamsShiftsConnectionTeamMap.md [hunk 7]
```diff
@@ -13,6 +13,8 @@ schema: 2.0.0
 
 ## SYNOPSIS
 
+**Note:** This cmdlet is currently in private preview.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsShiftsConnectionTeamMap.md:16]:
```suggestion
**Note:** This cmdlet is currently in public preview.
```

File: teams/teams-ps/teams/Remove-CsTeamsShiftsConnectionInstance.md [hunk 8]
```diff
@@ -13,6 +13,8 @@ schema: 2.0.0
 
 ## SYNOPSIS
 
+**Note:** This cmdlet is currently in private preview.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Remove-CsTeamsShiftsConnectionInstance.md:16]:
```suggestion
**Note:** This cmdlet is currently in public preview.
```

File: teams/teams-ps/teams/Remove-CsTeamsShiftsConnectionTeamMap.md [hunk 9]
```diff
@@ -13,6 +13,8 @@ schema: 2.0.0
 
 ## SYNOPSIS
 
+**Note:** This cmdlet is currently in private preview.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Remove-CsTeamsShiftsConnectionTeamMap.md:16]:
```suggestion
**Note:** This cmdlet is currently in public preview.
```

File: teams/teams-ps/teams/Remove-CsTeamsShiftsScheduleRecord.md [hunk 10]
```diff
@@ -13,6 +13,8 @@ schema: 2.0.0
 
 ## SYNOPSIS
 
+**Note:** This cmdlet is currently in private preview.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Remove-CsTeamsShiftsScheduleRecord.md:16]:
```suggestion
**Note:** This cmdlet is currently in public preview.
```

File: teams/teams-ps/teams/Set-CsTeamsShiftsConnectionInstance.md [hunk 11]
```diff
@@ -13,6 +13,8 @@ schema: 2.0.0
 
 ## SYNOPSIS
 
+**Note:** This cmdlet is currently in private preview.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsShiftsConnectionInstance.md:16]:
```suggestion
**Note:** This cmdlet is currently in public preview.
```

File: teams/teams-ps/teams/Test-CsTeamsShiftsConnectionValidate.md [hunk 12]
```diff
@@ -13,6 +13,8 @@ schema: 2.0.0
 
 ## SYNOPSIS
 
+**Note:** This cmdlet is currently in private preview.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Test-CsTeamsShiftsConnectionValidate.md:16]:
```suggestion
**Note:** This cmdlet is currently in public preview.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @bishalgautam-ms, kindly commit the suggested changes. Many thanks.

**Review by get-itips — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**